.github/workflows/reusable_build_sample_apps.yml

```diff
@@ -0,0 +1,148 @@
+name: Reusable build sample apps
+
+on:
+  workflow_call:
+    inputs:
+      use_latest_sdk_version:
+        description: "Whether this workflow should build sample apps with the latest SDK version"
+        type: boolean
+        required: false
+        default: false
+    secrets:
+      GOOGLE_CLOUD_MATCH_READONLY_SERVICE_ACCOUNT_B64:
+        required: true
+        description: "Base64-encoded service account for code signing (match) used by Fastlane."
+      FIREBASE_APP_DISTRIBUTION_SERVICE_ACCOUNT_CREDS_B64:
+        required: false
+        description: "Base64-encoded service account for Firebase App Distribution (optional)."
+      CUSTOMERIO_APN_WORKSPACE_SITE_ID:
+        required: true
+        description: "Secret for the APN sample app's Site ID in Customer.io."
+      CUSTOMERIO_APN_WORKSPACE_CDP_API_KEY:
+        required: true
+        description: "Secret for the APN sample app's CDP API key in Customer.io."
+      CUSTOMERIO_FCM_WORKSPACE_SITE_ID:
+        required: true
+        description: "Secret for the FCM sample app's Site ID in Customer.io."
+      CUSTOMERIO_FCM_WORKSPACE_CDP_API_KEY:
+        required: true
+        description: "Secret for the FCM sample app's CDP API key in Customer.io."
```

Comment:
I don't see them being reused from `inputs` directly. Can you please verify and delete if they are unused now?

Reply:
they are being forwarded to action